app/views/miq_ae_class/_class_instances.html.haml

```diff
@@ -1,3 +1,5 @@
+= react('AeInlineMethod', {:type => "aeInlineMethod"})
```

Comment:
this line will be removed and the component will be used as a part of https://github.com/ManageIQ/manageiq-ui-classic/pull/9047